department-of-veterans-affairs/abd-vro
# Add 25 minute TTL for all RabbitMQ messages

## What was the problem?

I could not find a default TTL for messages in RabbitMQ which could result in messages persisting longer than default termination grace period for k8s. 

Associated tickets or Slack threads:
- #2817 

## How does this fix it?[^1]

see title

## How to test this PR
- Confirmed by deployment of changes to `dev` env in LHDI

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
